README.md

```diff
@@ -1,32 +1,38 @@
 <p align="center">
 <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="img/rig_logo_dark.svg">
-    <source media="(prefers-color-scheme: light)" srcset="img/rig_logo.svg">
-    <img src="img/rig_logo.svg" style="width: 40%; height: 40%;" alt="Rig logo">
+    <source media="(prefers-color-scheme: dark)" srcset="img/rig-playgrounds-dark.svg">
+    <source media="(prefers-color-scheme: light)" srcset="img/rig-playgrounds-light.svg">
+    <img src="img/rig-playgrounds-light.svg" style="width: 40%; height: 40%;" alt="Rig logo">
 </picture>
 <br>
 <a href="https://crates.io/crates/rig-core"><img src="https://img.shields.io/crates/v/rig-core.svg" /></a>
 &nbsp;
+<a href="https://crates.io/crates/rig-core"><img src="https://img.shields.io/crates/d/rig-core?color=orange" /></a>
+&nbsp;
 <a href="https://discord.gg/playgrounds"><img src="https://img.shields.io/discord/511303648119226382?color=%236d82cc&label=Discord&logo=discord&logoColor=white" /></a>
 &nbsp;
+<a href="https://github.com/0xPlaygrounds/rig"><img src="https://img.shields.io/github/stars/0xPlaygrounds/rig?style=social" alt="stars - rig" /></a>
+<br>
 <a href=""><img src="https://img.shields.io/badge/built_with-Rust-dca282.svg?logo=rust" /></a>
 &nbsp;
-<a href="https://twitter.com/Playgrounds0x"><img src="https://img.shields.io/badge/Twitter-Follow_Us-1d9bf0.svg?style=flat /"></a>
-&nbsp;
-<a href="https://github.com/0xPlaygrounds/rig?tab=MIT-1-ov-file"><img src="https://img.shields.io/badge/license-MIT-00bfff.svg?color=red" /></a>
+<a href="https://twitter.com/Playgrounds0x"><img src="https://img.shields.io/twitter/follow/Playgrounds0x"></a>
+<a href="https://docs.rs/rig-core/latest/rig/"><img src="https://img.shields.io/badge/📖documentation-docs.rs-dca282.svg" /></a>
+<br>
 </p>
 &nbsp;
```

Comment:
<img width="840" alt="image" src="https://github.com/user-attachments/assets/f55908ad-002f-4abd-8e5a-92b2532dd858">
This seems wrong / duplicates.., might want to clean this up